ENGAGEHF/Bluetooth/Views/MeasurementRecordedView.swift

```diff
@@ -0,0 +1,183 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+@_spi(TestingSupport)
+import SpeziBluetooth
+import BluetoothServices
```

Comment:
Note on my added code for you @nriedman: Required for to the manual injection in the Bluetooth service. Not really a public API but part of the testing support and we can utilize this here in the Preview. Maybe @Supereg has some further inputs here and/or this is something to look at in SpeziBluetooth.